src/Raven.Client/Documents/Session/ClusterTransactionSession.cs

```diff
@@ -494,6 +523,8 @@ public interface IClusterTransactionOperations : IClusterTransactionOperationsBa
 
         Dictionary<string, CompareExchangeValue<T>> GetCompareExchangeValues<T>(string[] keys);
 
+        Dictionary<string, CompareExchangeValue<T>> GetCompareExchangeValues<T>(string startWith, int? start = null, int? pageSize = null);
```

Comment:
startWith => startsWith
int start = 0;
int pageSize = int.MaxValue

Reply:
Maybe
int start = 0,
int pageSize = 25

(like in `LoadStartingWith`)

Reply:
ok

Reply:
done